R/rows-db.R

```diff
@@ -63,9 +74,9 @@ rows_insert.tbl_dbi <- function(x, y, by = NULL, ...,
     }
 
     con <- dbplyr::remote_con(x)
-    sql <- sql_rows_insert(x, y)
-    dbExecute(con, sql, immediate = TRUE)
-    invisible(x)
+    sql <- sql_rows_insert(x, y, returning = returning)
```

Comment:
Do we really need to pass `returning` to the `sql_()` methods?

```suggestion
    sql <- paste0(
      sql_rows_insert(x, y, returning = returning)
      if (!is.null(returning)) paste0("\n", sql_returning(...))
    )
```

Reply:
If `returning` is not an argument of the `sql_()` methods then I would rather have something like

```r
sql_rows_insert(x, y, returning = returning) %>%
  sql_add_returning(x, returning)
```

and export `sql_add_returning()` (this should be exported to complete the other sql functions). To me this feels like a more user-friendly interface. What do you think?

Reply:
I think we need an `sql_returning()` generic anyway -- to special-case both the clause and the way column expressions are constructed (to work around that RSQLite bug). I don't mind `sql_add_returning()` as an equivalent to the `paste0()` call in my proposal.